Tests/RxCocoaTests/UITextField+RxTests.swift

```diff
@@ -13,35 +13,46 @@ import XCTest
 
 // UITextField
 final class UITextFieldTests : RxTest {
+    
     func test_TextCompletesOnDealloc() {
         ensurePropertyDeallocated({ UITextField() }, "a", comparer: { $0 == $1 }) { (view: UITextField) in view.rx.text }
     }
     func test_ValueCompletesOnDealloc() {
         ensurePropertyDeallocated({ UITextField() }, "a", comparer: { $0 == $1 }) { (view: UITextField) in view.rx.value }
     }
-
+    
     func testSettingTextDoesntClearMarkedText() {
         let textField = UITextFieldSubclass(frame: CGRect.zero)
-
+        
         textField.text = "Text1"
-        textField.set = false
+        textField.settedText = false
         textField.rx.text.on(.next("Text1"))
-        XCTAssertTrue(!textField.set)
+        XCTAssertTrue(!textField.settedText)
         textField.rx.text.on(.next("Text2"))
-        XCTAssertTrue(textField.set)
+        XCTAssertTrue(textField.settedText)
+    }
+    
+    func testLabel_attributedTextObserver() {
```

Comment:
I think this is copied code from `UILabel`, but nothing is updated, it doesn't test `UITextField`.